Assignment3/test.js

```diff
@@ -0,0 +1,50 @@
+const array = [];
+array[0] = [3, 56, 7, 32, 9, 14];
+array[1] = [21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 27, 56, 59];
+array[2] = [21, 87, 91, 31, 97, 59, 8, 83, 27, 23, 76, 72, 31, 91, 65, 77, 84, 92, 56, 59];
+array[3] = [111, 0, 20, 0, 2];
```

Comment:
yet again 5 statements to initialize an array. use multi-line array declaration instead.